docs/core/deploying/runtime-package-store.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Runtime package store | Microsoft Docs
+description: This topic explains the runtime package store and target manifests used by .NET Core.
+keywords: .NET, .NET Core, dotnet store, runtime package store
+author: bleroy
+ms.author: mairaw
+ms.date: 06/26/2017
+ms.topic: article
+ms.prod: .net-core
+ms.devlang: dotnet
+ms.assetid: 9521d8b4-25fc-412b-a65b-4c975ebf6bfd
+---
+
+# Runtime package store
+
+[!INCLUDE [core-preview-warning](~/includes/core-preview-warning.md)]
+
+Starting with .NET Core 2.0, it's possible to package and deploy apps against a known set of packages that exist in the target environment. The benefits are faster deployments, lower disk space use, and improved startup performance in some cases.
+
+This feature is implemented as a *runtime package store*, which is a directory on disk next to the the [`dotnet` host](../tools/dotnet.md) where packages are stored (typically at *.dotnet/store* in the user's profile). Under this directory, there are subdirectories for architectures and [target frameworks](../../standard/frameworks.md). The file layout is similar to the way that [NuGet assets are laid out on disk](/nuget/create-packages/supporting-multiple-target-frameworks#framework-version-folder-structure):
```

Comment:
Yes, that's correct. I fixed that line. The main thrust of the topic is the user-local one, and I don't want to confuse anyone by mentioning the presence of the installed store at this point in the topic. WRT the original text of this topic, they may have moved it at some point ... hard to say.

I think my fix tho is to not discuss the less-relevant store at all until the section *ASP.NET Core implicit store* further down the topic. Even then, I don't want to get too much into the weeds on it. I think your other point about SCD is *very* important to get into that section, and I'm adding that now.